capable/Cargo.toml

```
@@ -1,15 +1,23 @@
 [package]
 name = "mc-sgx-capable"
+# When updating version, update the deps.rs link in README.md
 version = "0.1.0"
 edition = "2021"
 authors = ["MobileCoin"]
 rust-version = "1.62.1"
 license = "Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/mobilecoinfoundation/sgx"
-description = '''
-This crate exposes an idiomatic rust interface to the SGX SDK's `libsgx_capable` libraries.
-'''
+description = "Rust wrappers for the `sgx_capable` library."
+
+categories = ["external-ffi-bindings", "hardware-support"]
+description = "FFI linkage for the `sgx_capable` library."
+edition = "2021"
+keywords = ["sgx"]
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/mobilecoinfoundation/sgx"
+rust-version = "1.62.1"
```

:upside_down_face: Looks like most of these are repeats.  `capable` is not in the root `Cargo.toml` workspace, if it gets added then these duplicates show up as cargo errors.
```suggestion
keywords = ["sgx"]
```